app/assets/stylesheets/styles/ui/inputs/s3_upload.css.scss

```diff
@@ -0,0 +1,79 @@
+/**
+ * A class that covers all the s3 upload buttons.
+ */
+
+.s3-upload-input{
+  .click-to-upload-image{
+    @include add-new-button();
+    @include box-shadow(none);
+    @include font-awesome-mixin();
+
+    cursor: pointer;
+    color: $openfarm-green-muted;
+    width: 100px;
+    height: 100px;
+    border-radius: 1em;
+    display: inline-block;
+    margin-bottom: 1rem;
+    position: relative;
+    vertical-align: top;
+
+    &:hover {
+      color: darken($of-green, 10%);
+    }
+
+    &:before{
+      content: "\f06c";
+      font-size: 4rem;
+      position: absolute;
+      z-index: 0;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+    }
+
+    .wrapper{
+      height: 100%;
+    }
+
+    img.no-image{
+      transform: translate(-50%, -50%);
+      position: relative;
+      top: 50%;
+      left: 50%;
+    }
+
+    img{
```

Comment:
Merge rule `img` with rule on line 39<br>Opening curly brace `{` should be preceded by one space